packages/db-objects-redis/src/lib/objects/objectsUtils.ts

```diff
@@ -574,12 +579,18 @@ export function sanitizePath(id: string, name: string) {
     return { id: id, name: name };
 }
 
-export function checkObject(obj: ioBroker.Object, options: Record<string, any>, flag: any) {
+export function checkObject(
+    obj: ioBroker.Object,
+    options: Record<string, any>,
+    flag: CONSTS.GenericAccessFlags
+): boolean {
     // read rights of object
     if (!obj || !obj.common || !obj.acl || flag === CONSTS.ACCESS_LIST) {
         return true;
     }
 
+    // FIXME: what if flag is ACCESS_DELETE or ACCESS_CREATE?
+
```

Comment:
We currently don't handle this case, but its theoretically possible to call the method this way (I think we already do!)